### ISSUE TITLE
Introduce a command to request an API token

### ISSUE THREAD

**Issue by gastaldi** [minishift/minishift#567]
The following command echoes the URL to request an API token in the console:

```
echo `minishift console --url`/oauth/token/request
```
It would be nice if minishift had a command to automatically open that in the browser (similar to what `minishift console` does).

**Comment by praveenkumar** [comment]
@hferentschik Can we put a milestone, looks like trivial to do something like

```
minishift console --auth
```

**Comment by hferentschik** [comment]
+1

**Comment by gastaldi** [comment]
Cool! And `minishift console --auth --url` would display the URL right?

**Comment by praveenkumar** [comment]
> minishift console --auth --url would display the URL right?

No, `minishift console --url` will display the url.

**Comment by gastaldi** [comment]
Works for me

**Comment by gastaldi** [comment]
Since this URL is meant to generate an OAuth token, perhaps the param should be named `--request-token`? `--auth` looks like it should redirect to the user/pass login screen IMHO. Thoughts?

**Comment by hferentschik** [comment]
Sure, makes sense. We can discuss the exact name when we actually implement the feature.

**Comment by LalatenduMohanty** [comment]
Fixed with #1181
